audit/projects/k8s-infra-e2e-boskos-scale-02/buckets/artifacts.k8s-infra-e2e-boskos-scale-02.appspot.com/iam.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "projectEditor:k8s-infra-e2e-boskos-scale-02",
```

Comment:
The `artifacts.k8s-infra-e2e-boskos-scale-xx.appspot.com` noise is from CI jobs auto-creating GCR buckets on first use.  We could eliminate this by pre-creating these buckets: https://github.com/kubernetes/k8s.io/issues/1999